### PR TITLE
Add kurtzumbahlen recommendations

### LINKED ISSUE #6: kurtzumbahlen checklist
- [x] Create a branch
- [x] Add a file
- [ ] Make a PR
- [ ] Merge

:ramen:

### PR DESCRIPTION
**Pull request by kurtzumbahlen**
Adding recommendations for St. Paul/Minneapolis 

Closes #6

### CODE REVIEW

File: kurtzumbahlen-rec.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# St. Paul, MN
+
+### Places to Eat
+- Surly for :beer: 
+- Tori Ramen for ramen
```
Comment (kyle-kluever) [inline kurtzumbahlen-rec.md]: Where is Tori Ramen?

File: kurtzumbahlen-rec.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# St. Paul, MN
+
+### Places to Eat
```
Comment (kyle-kluever) [inline kurtzumbahlen-rec.md:3]: Where are the emojis? :japanese_castle:  :japanese_goblin:  :kazakhstan:  :pager:

**Review by kyle-kluever — APPROVED** [review]
Stay woke

File: kurtzumbahlen-rec.md (hunk 2 above)
Comment (kurtzumbahlen) [reply kurtzumbahlen-rec.md:3]: :zipper_mouth_face:

**Comment by kyle-kluever** [comment]
@kurtzumbahlen Check it out

**Comment by kurtzumbahlen** [comment]
@kyle-kluever  thanks for the in-depth reviews!

**Pull request merged**